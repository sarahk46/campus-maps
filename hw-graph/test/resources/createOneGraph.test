# Tests that a graph can be created.

# Create a graph
CreateGraph graph1