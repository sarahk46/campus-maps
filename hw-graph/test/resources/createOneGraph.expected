# Tests that a graph can be created.

# Create a graph
created graph graph1